src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -21,10 +21,17 @@ public class ViewCommand extends Command {
     public static final String MESSAGE_NO_SUCH_TELEGRAM =
             "There is no one in your address book with the telegram handle: @";
 
+    public static final String MESSAGE_TOO_SHORT_TELEGRAM =
+            "The telegram handle entered is too short (min 5 characters).";
+
+    public static final String MESSAGE_ILLEGAL_CHARACTERS_IN_TELEGRAM =
+            "The telegram handle entered contains illegal characters.\n"
+            + "Telegram handles should only contain letters, numbers and underscores.";
+
```

Comment:
Just a suggestion, it may be better to move general error messages to `seedu/address/logic/Messages.java` file.